tests/generate.rs

```diff
@@ -138,3 +138,34 @@ fn generate_github(#[case] remote: &str) {
         read_to_string(temp_path.join("knope.toml")).unwrap(),
     );
 }
+
+/// Run `--generate` on a repo with a GitHub remote.
```

Comment:
```suggestion
/// Run `--generate` on a repo with a Codeberg remote.
```

Reply:
oh no, the copy paste has been exposed xD